manage.py

```diff
@@ -140,6 +140,48 @@ def _add_live_test_subparser(subparsers):
     )
 
 
+def _add_validate_schema_subparser(subparsers):
```

Comment:
Non-public mehods/functions don't actually _need_ docstrings so it wouldn't complain here.. but I didn't realize this was missing one so I'll add it for consistency